Assets/Mirror/Runtime/NetworkTime.cs

```diff
@@ -45,16 +45,11 @@ public static void Reset()
             offsetMax = double.MaxValue;
         }
 
-        internal static NetworkPingMessage GetPing()
```

Comment:
Make it obsolete in case people are using it.

Reply:
Its internal, nobody could be using it.